frontend/src/pages/Graph/GraphToolbar/GraphFindPF.tsx

```diff
@@ -0,0 +1,1118 @@
+import * as React from 'react';
+import { Button, Tooltip, ButtonVariant, TextInput, Form } from '@patternfly/react-core';
+import { Controller, Graph, GraphElement } from '@patternfly/react-topology';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { KialiAppState } from '../../../store/Store';
+import { findValueSelector, hideValueSelector, edgeLabelsSelector, edgeModeSelector } from '../../../store/Selectors';
+import { GraphToolbarActions } from '../../../actions/GraphToolbarActions';
+import GraphHelpFind from '../../../pages/Graph/GraphHelpFind';
+import { CyNode, CyEdge } from '../../../components/CytoscapeGraph/CytoscapeGraphUtils';
```

Comment:
`CyNode` and `CyEdge` are a bunch of string constants for names of data attributes that are stored on Nodes and Edges.  Both Cy and PF allow you to add app-specific "Data" to your elements, basically as an `any` type.  Sometimes we need to refer to those fields by name, and to reduce the chance of typos I have all of these constants.   Unfortunately the current names include "Cy" and defined in CytoscapeGraphUtil.ts.  I guess now is a good time to change it because you are right, it's not great to see "Cy" in the PF impl.  I was just avoiding it because I couldn't think of a good, short name to replace `CyNode` and `CyEdge`.   I'll change it to `NodeAttr` and `EdgeAttr`, longer but ok I guess.

Note that we use the same data attributes in both impls.  In fact the PF graph uses the same JSON that we send from the backend.  That JSON is configured well for Cytoscape, but I just adapt it to be loaded into PF.